### PR TITLE
[Stewart] Add alignment rules

### PR DESCRIPTION
**Pull request by kjellr**
This PR adds the Twenty Twenty-Two alignment rules to Stewart. Previously, there were no alignment rules here at all.

### CODE REVIEW

File: stewart/theme.json [hunk 1]
```diff
@@ -105,6 +105,9 @@
 			"margin": {
 				"horizontal": "30px",
 				"vertical": "30px"
+			},
+			"spacing": {
+				"outer": "min(1em, 30px)"
```
Comment (scruffian) [inline stewart/theme.json]: TIL

Comment (scruffian) [reply stewart/theme.json]: Does this actually do anything?

Comment (kjellr) [reply stewart/theme.json]: Ha, whoops. No, I meant to put a `vw` unit there instead of the `1em`. This should scale more nicely now.

Comment (scruffian) [reply stewart/theme.json]: I still don't know what `spacing > outer` does though!

Comment (kjellr) [reply stewart/theme.json]: Oh! That's the variable used in all the new layout CSS here: 

https://github.com/Automattic/themes/blob/fafc2c5119fabffa2cddf78ce7112f65a1e884c9/stewart/style.css#L141-L142

It defines the outer left and right padding for the site. It's the same variable name [used in Twenty Twenty-Two](https://github.com/WordPress/twentytwentytwo/blob/0411debe4f66661d878c171d98614d8cfd0d4a5e/theme.json#L144) for easy copy/paste if this code needs to update.

Comment (scruffian) [reply stewart/theme.json]: Ah nice one.

**Review by scruffian — APPROVED** [review]
LGTM

**Pull request merged**